### PR TITLE
InterPride AGM page

### PR DESCRIPTION
**Pull request by boyeborg**

### CODE REVIEW

**Comment by boyeborg** [comment]
https://oslopride-website-dotjtuxjw.now.sh/interpride2020

File: pages/interpride2020/index.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+  if (interPride.status !== "SUCCESS") {
+    // TODO: Make a better UX while loading
+    return <div>Laster ...</div>;
+  }
+
+  const { body, preamble, image } = interPride.data;
+
+  return (
+    <>
```
Comment (siddise) [inline pages/interpride2020/index.jsx]: Skal dette være her?

File: pages/interpride2020/index.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            url: "https://oslopride.no/interpride2020",
+            locale: "en_US",
+            site_name: "Oslo Pride",
+            title: "InterPride AGM 2020",
+            description: preamble,
+            images: [{ url: imageUrlFor(image).url() }]
+          }
+        }}
+      />
+    </>
```
Comment (siddise) [inline pages/interpride2020/index.jsx]: Skal dette være her?

File: pages/interpride2020/index.jsx (hunk 1 above)
Comment (boyeborg) [reply pages/interpride2020/index.jsx]: jupp!

File: pages/interpride2020/index.jsx (hunk 2 above)
Comment (boyeborg) [reply pages/interpride2020/index.jsx]: Jupp!

File: pages/interpride2020/index.jsx (hunk 1 above)
Comment (boyeborg) [reply pages/interpride2020/index.jsx]: https://reactjs.org/docs/fragments.html#short-syntax

File: pages/interpride2020/index.jsx (hunk 2 above)
Comment (boyeborg) [reply pages/interpride2020/index.jsx]: https://reactjs.org/docs/fragments.html#short-syntax

**Pull request merged**